multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -24,20 +31,672 @@ use node_lib::database::db_initializer::{
+        exp_final_cons_node_service_fee_balance_minor: 0,
+        // This account was granted with the full size as its lowest balance from the set makes
+        // it weight the most
+        exp_final_service_fee_balance_serv_node_1_minor: owed_to_serv_node_1_minor,
+        exp_final_service_fee_balance_serv_node_2_minor: owed_to_serv_node_2_minor
+            - gwei_to_wei::<u128, u64>(2_345_678),
+        // This account dropped out from the payment, so received no money
+        exp_final_service_fee_balance_serv_node_3_minor: 0,
+    };
+
+    let process_scan_request_to_node =
+        |real_node: &MASQRealNode, ui_port: u16, scan_type: ScanType, context_id: u64| {
+            let ui_client = real_node.make_ui(ui_port);
+            ui_client.send_request(UiScanRequest { scan_type }.tmb(context_id));
+            let response = ui_client.wait_for_response(context_id, Duration::from_secs(10));
+            UiScanResponse::fmb(response).unwrap();
+        };
+
+    let stimulate_payments =
+        |_cluster: &mut MASQNodeCluster,
+         real_consuming_node: &MASQRealNode,
+         _global_test_config: &TestInputsOutputsConfig| {
+            process_scan_request_to_node(
+                &real_consuming_node,
+                consuming_node_ui_port,
+                ScanType::Payables,
+                1111,
+            )
+        };
+
+    let start_serving_nodes_and_run_check = |cluster: &mut MASQNodeCluster,
+                                             serving_node_1_attributes: ServingNodeAttributes,
+                                             serving_node_2_attributes: ServingNodeAttributes,
+                                             serving_node_3_attributes: ServingNodeAttributes,
+                                             test_global_config: &TestInputsOutputsConfig|
+     -> (MASQRealNode, MASQRealNode, MASQRealNode) {
+        let ports = test_global_config.ui_ports_opt.as_ref().unwrap();
+        let mut vec: Vec<MASQRealNode> = vec![
+            (serving_node_1_attributes, ports.serving_node_1, 2222),
+            (serving_node_2_attributes, ports.serving_node_2, 3333),
+            (serving_node_3_attributes, ports.serving_node_3, 4444),
+        ]
+        .into_iter()
+        .map(|(serving_node_attributes, ui_port, context_id)| {
+            let serving_node = cluster.start_named_real_node(
+                &serving_node_attributes.name,
+                serving_node_attributes.index,
+                serving_node_attributes.config,
+            );
+
+            process_scan_request_to_node(&serving_node, ui_port, ScanType::Receivables, context_id);
+
+            serving_node
+        })
+        .collect();
+        (vec.remove(0), vec.remove(0), vec.remove(0))
+    };
+
+    test_body(
+        test_global_config,
+        stimulate_payments,
+        start_serving_nodes_and_run_check,
+    );
+}
+
+fn make_init_config(chain: Chain) -> DbInitializationConfig {
```

Comment:
A suggestion that you wanted me to add:

`make_init_config()` -> `make_db_init_config()`